src/components/Card/Card.scss

```diff
@@ -71,6 +71,14 @@
   }
 }
 
+.Subsection {
+  + .Subsection {
+    margin-top: spacing();
+    padding-top: spacing();
+    border-top: border-width() solid color('sky');
```

Comment:
is `border-width() solid color('sky')` effectively the same thing as `border()` ?

Reply:
☝️ Yes, that's exactly what the default/base `border()` resolves to.
```suggestion
    border-top: border();
```

Reply:
Agreed this should just be `border()`
```suggestion
    border-top: border();
```